infra/endpoints.tf

```diff
@@ -226,6 +225,13 @@ resource "aws_vpc_endpoint" "kinesis_stream_endpoint" {
   }
 }
 
+# TODO - remove and use the key we manage in this Terraform workspace
+data "aws_kms_key" "existing_lambda_env_encryption" {
```

Comment:
Seemed safest to add this in, but I'm not actually sure it's even used. The Lambda functions all seem to use the default KMS key for env var encryption.